### DISCUSSION TITLE
Redirection error: supabase does not give permission for other routes except login

### DISCUSSION THREAD

**Discussion by arslanBekbolotov** [supabase/supabase#20110] [Questions]
I recently created an application with nuxt and nuxtjs@supabase and I have a problem until I log in I can not go through the pages.I checked that without supabase everything works in it but do not know how to solve this problem
[dashboard.zip](https://github.com/supabase/supabase/files/13812618/dashboard.zip)

**Comment by AyushNigam11** [comment]
If you're using the prefabricated router from supabase the redirect request should go to /auth/callback, so supabase can handle the authorization code.

Just add the corrent redirect url to your signInWithOAuth() function call, like this:

let { data, error } = await supabase.auth.signInWithOAuth({
      provider: 'google',
      options: {
        redirectTo: window.location.origin + "/auth/callback"
      }
})
